src/instrumentsscene/qml/MuseScore/InstrumentsScene/internal/LayoutPanelItemDelegate.qml

```diff
@@ -216,6 +216,7 @@ FocusableControl {
         spacing: 2
 
         VisibilityBox {
+            visible: root.type !== LayoutPanelItemType.SYSTEM_OBJECTS_LAYER
```

Comment:
Since this button is now hidden for system object layers, could you please remove this function -- SystemObjectsLayerTreeItem::onVisibleChange?